sbt_common/display/src/main/scala/uk/ac/wellcome/display/models/DisplayWork.scala

```diff
@@ -24,6 +24,9 @@ case class DisplayWork(
     dataType = "String",
     value = "A description of specific physical characteristics of the work.") physicalDescription: Option[
     String] = None,
+  @ApiModelProperty(
+    dataType = "List[uk.ac.wellcome.display.models.DisplayWorkType]",
+    value = "The type of work.") workType: Option[DisplayWorkType] = None,
```

Comment:
Is this what it says in the ontology?

Reply:
No. I couldn't find the correct reference text, hoping @jtweed can enlighten.

Reply:
Found it: https://github.com/wellcometrust/platform/blob/f3217ea06b7283ffdba3b18d66a57fd26bf57871/ontologies/Schema/works-ontology.ttl

Reply:
Yep:

https://github.com/wellcometrust/platform/blob/7332db7cc3f5017cf2e41cba39f25b61d08d0719/ontologies/Schema/works-ontology.ttl#L28